### PR TITLE
[3.x] fix provider avatar feature

### PR DESCRIPTION
**Pull request by Sairahcaz**

### CODE REVIEW

File: src/SetsProfilePhotoFromUrl.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ trait SetsProfilePhotoFromUrl
     public function setProfilePhotoFromUrl(string $url)
     {
         $name = pathinfo($url)['basename'];
-        file_put_contents($file = sys_get_temp_dir().Str::uuid()->toString(), file_get_contents($url));
+        file_put_contents($file = sys_get_temp_dir().'/'.Str::uuid()->toString(), file_get_contents($url));
```
Comment (tegansnyder) [inline src/SetsProfilePhotoFromUrl.php:19]: YES! I'm getting this very issue now.
image

**Comment by james-wong123** [comment]
hi' i am new to laravel. is there a way to customize the controller without changing the vendor folder

**Comment by Atiladanvi** [comment]
Plan to merge ?

**Comment by james-wong123** [comment]
> 

ya

**Review by joelbutcher — APPROVED** [review]

**Pull request merged**